src/ui/state/app.ts

```diff
@@ -75,6 +75,7 @@ export interface LoadedRegions {
   loading: TimeStampedPointRange[];
   loaded: TimeStampedPointRange[];
   indexed: TimeStampedPointRange[];
+  indexedAndLoaded: TimeStampedPointRange[];
```

Comment:
Lets leave a TODO here that we want to move (`loading`,`indexed`, `loaded`) into a protocol field and rename `indexedAndLoaded` to `loading` and use a more relevant `loaded`

Reply:
can `indexedAndLoaded` be a selector?

Reply:
I'm not so sure on moving `indexedAndLoaded` to `loading`, I think those two are closer to `loaded`, and the protocol definition of `loading` is probably fine? (Maybe when we add soft-focus `loading` would become `focusedAndProtocolLoading` or something?

Reply:
oh yeah, sorry, `indexedAndLoaded` **is** a selector now, and I just forgot to update the usage. Good catch 👍